pkg/receive/handler.go

```diff
@@ -1240,11 +1240,6 @@ func (p *peerWorker) RemoteWriteAsync(ctx context.Context, req *storepb.WriteReq
 
 		tracing.DoInSpan(ctx, "receive_forward", func(ctx context.Context) {
 			_, err := storepb.NewWriteableStoreClient(p.cc).RemoteWrite(ctx, req)
-			if isConflict(err) && p.wp.Size() > 1 {
```

Comment:
we can't simply shallow this error code due to other error causes, expose it for now, need to revisit this later.